applications/dashboard/views/email/src/email-basic.php

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <!--<![endif]-->
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <style>
+    <style type="text/css">
```

Comment:
This one isn't necessary. This file gets compiled into a Smarty template (`email/email-basic.tpl`) where this block of css gets inlined and the block disappears. The `email/email-basic.tpl` file is the actual email template that gets used.